controllers/app.controllers.js

```diff
@@ -83,3 +84,14 @@ exports.patchArticleVotesById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeCommentById(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
A nice alternative to this is using the `res.sendStatus(204)` which looks a little nicer than an empty .send() added to the end